santa-tracker-base.conf

```
@@ -48,36 +44,28 @@ noOptimizations = ${base} {
     # apply-android-resource-value-change-to = ${resource-to-change}
 }
 
-onlyVfsRetention = ${noOptimizations} {
-    title = "2 – VFS retention"
-    gradle-args = ${noOptimizations.gradle-args} ${enableVfsRetention}
+onlyFileWatching = ${noOptimizations} {
+    title = "2 – File watching"
```

```suggestion
    title = "2 – FS watching"
```

I think we don't want to use `File watching`.